metricq/client.py

```diff
@@ -121,7 +123,9 @@ async def get_metrics(
         :param limit: limit the number of results to return
         :return: either a {name: metadata} dict (metadata=True) or a list of metric names (metadata=False)
         """
-        arguments = {"format": "object" if metadata else "array"}
+        arguments: Dict[str, Union[str, Sequence[str], bool, float]] = {
```

Comment:
I would actually go with a generic "kwargs"-Type rather than the specific types we put in here at the moment.

```suggestion
        arguments: Dict[str, Any] = {
```